StatusDocumentation.md

```diff
@@ -57,11 +81,15 @@ The run loop sync resource tracks [run loops](https://developer.apple.com/docume
 
 During the normal lifecycle of an app, its run loops normally switch often between busy and idle states, and no special significance should be paid to a busy run loop, as it is usually accompanied by other busy sync resources, which better describe what the system is doing.
 
-A typical idle status response:
+A typical busy resource representation:
 
 ```
-Run Loop
-⏱ “Main Run Loop”
+{
+	"name": "run_loop",
+	"description": {
+		"name": "Main Run Loop"
+	}
+}
 ```
 
 ### One-time Event
```

Comment:
Agree, also, as discussed, we should aggregate all network requests that occur in parallel, and log them under the same resource.

However since the scope of this change is quite large, I suggest we should do the changes in this order:
1. Finish with this PR, and by this keep it for now as "One-time event" in iOS.
2. Finish the logs-format-unification changes in the Android & JS parts.
3. Do the network requests resource changes in iOS, and in Android also if needed, to achieve in both sides the same network logs (list of occurring network requests [URLs]).